packages/flutter/lib/src/widgets/animated_switcher.dart

```diff
@@ -84,30 +85,33 @@ typedef Widget AnimatedSwitcherLayoutBuilder(List<Widget> children);
 ///   @override
 ///   Widget build(BuildContext context) {
 ///     return new Material(
-///       child: Column(
-///         mainAxisAlignment: MainAxisAlignment.center,
-///         children: <Widget>[
-///           new AnimatedSwitcher(
-///             duration: const Duration(milliseconds: 200),
-///             transitionBuilder: (Widget child, Animation<double> animation) {
-///               return new ScaleTransition(child: child, scale: animation);
-///             },
-///             child: new Text(
-///               '$_count',
-///               // Must have this key to build a unique widget when _count changes.
-///               key: new ValueKey<int>(_count),
-///               textScaleFactor: 3.0,
+///       child: new Directionality(
+///         textDirection: TextDirection.rtl,
+///         child: Column(
+///           mainAxisAlignment: MainAxisAlignment.center,
+///           children: <Widget>[
+///             new AnimatedSwitcher(
+///               duration: const Duration(milliseconds: 500),
+///               transitionBuilder: (Widget child, Animation<double> animation) {
+///                 return new ScaleTransition(child: child, scale: animation);
+///               },
+///               child: new Text(
+///                 '$_count',
+///                 // Must have this key to build a unique widget when _count changes.
+///                 key: new ValueKey<int>(_count),
+///                 textScaleFactor: 3.0,
+///               ),
 ///             ),
-///           ),
-///           new RaisedButton(
-///             child: new Text('Click!'),
-///             onPressed: () {
-///               setState(() {
-///                 _count += 1;
-///               });
-///             },
-///           ),
-///         ],
+///             new RaisedButton(
+///               child: new Text('Click!'),
```

Comment:
accessibility folks will tell you to use a more descriptive label, like "Increment"